setup/www/resources/config/nodejs.org

```diff
@@ -40,7 +40,11 @@ server {
     default_type text/plain;
     index index.html;
 
+    error_page 404 @localized_404;
+
     location / {
+        try_files $uri $uri/ @english_fallback;
```

Comment:
@rvagg `try_files $uri $uri/` is really what you would expect from a standard static server, and probably the most used nginx `try_files` pattern.

1. Try matching the current request URI as a filename: `https://nodejs.org/en/index.html` will match  `/home/www/nodejs/en/index.html` directly, no need to try the second `try_files` option.
2. If a file wasn't matched, try the URI as a directory instead: `https://nodejs.org/en/about` doesn't match a filename directly, meaning nginx will try `/en/about/` where it finds `/home/www/nodejs/en/about/index.html`.

The third option provides as fallback if none of the above matched something in the filesystem. Really the essence of what we're trying to do with this PR; try to find an english version of the requested file, instead of serving a 404 directly.